src/core/realm.js

```diff
@@ -0,0 +1,20 @@
+import Realm from 'realm';
+import {Account, TokenPrice, Transaction} from './realm-schemas';
+
+let realm;
+
+export async function initRealm() {
+  realm = await Realm.open({
+    path: 'dock',
+    schema: [TokenPrice, Transaction, Account],
+    schemaVersion: 3,
+    deleteRealmIfMigrationNeeded: false,
+    migration: () => {
+      // No migration required so far
```

Comment:
we'll want to give this a think about how we want to do migrations when it comes up.

Reply:
Yes, it will be required before we deploy this new version. I will add this in an additional PR this week.